pkgs/applications/audio/snapcast/default.nix

```diff
@@ -9,24 +9,15 @@ assert pulseaudioSupport -> libpulseaudio != null;
 
 stdenv.mkDerivation rec {
   pname = "snapcast";
-  version = "0.27.0";
+  version = "0.28.0";
 
   src = fetchFromGitHub {
     owner  = "badaix";
     repo   = "snapcast";
     rev    = "v${version}";
-    sha256 = "sha256-dlK1xQQqst4VQjioC7MZzqXwMC+JfqtvnD5lrOqGhYI=";
+    sha256 = "sha256-XxpiLojs1TI3qM7yXS+OPcHKo6pLcfJ+Eop7GjQ4HQw=";
   };
 
-  patches = [
-    # Can be removed with next release after 0.27.0
-    (fetchpatch {
```

Comment:
Please remove the unused fetchpatch from inputs.

Reply:
fixed in https://github.com/NixOS/nixpkgs/pull/305661/commits/5c4361ae2fc982934e13dd30983fdc7764e0a1ca